index.php

```diff
@@ -2,19 +2,24 @@
 <!DOCTYPE html>
 <html lang="en">
 <?php require_once __DIR__ .'/template/head.phtml';?>
-<title><?=$titleIndex?></title>
+<?php
+$title = $page_titles[basename($_SERVER['SCRIPT_FILENAME'])];

```

Comment:
@ReginaBorisova 
It looks that it would be better to have a function to get a page title